Mathlib/Analysis/Normed/Group/Constructions.lean

```diff
@@ -229,6 +229,8 @@ instance Prod.toNorm : Norm (E × F) where norm x := ‖x.1‖ ⊔ ‖x.2‖
 
 lemma Prod.norm_def (x : E × F) : ‖x‖ = max ‖x.1‖ ‖x.2‖ := rfl
 
+@[simp] lemma Prod.norm_mk (x : E) (y : F) : ‖(x, y)‖ = max ‖x‖ ‖y‖ := rfl
```

Comment:
Given that `Prod.norm_def` isn't a `simp` lemma, are you sure that this is a good `simp` lemma?

Reply:
I think yes, because it matches `Norm.norm (Prod.mk _ _)` only, not any `Norm.norm (_ : Prod _ _)`.